lib/validators/movie-list.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+const Joi = require('joi');
+
+module.exports = Joi.object().keys({
+  title: Joi.string().min(1).max(255).optional(),
+  release_year: Joi.object().keys({
+    eq: Joi.number().integer().min(1878).max(9999).optional(),
+    gt: Joi.number().integer().min(1878).max(9999).optional(),
+    lt: Joi.number().integer().min(1878).max(9999).optional()
+  })
```

Comment:
Do you think it's necessary to have the `min` and `max` restrictions? These would be parameters for a read query, so if a consumer tried to search for a movie in 1400, we could just return a `200 OK` without much risk to the database instead of rejecting the request outright with a `422 Unprocessable Entity`.